src/utils/documentExportHelper.ts

```diff
@@ -44,22 +51,32 @@ export const getDownloadLink = async (
             } else {
                 engineVersion = (documentResponse.parsedData as unknown as { engineVersion: string })?.engineVersion;
             }
-            generateExportUrl += `&engineVersion=${engineVersion}`;
-            if (engineCommitSha) generateExportUrl += `-${engineCommitSha}`;
+
+            if (format !== DownloadFormats.PDF) {
+                generateExportUrl += `&engineVersion=${engineVersion}`;
+                if (engineCommitSha) generateExportUrl += `-${engineCommitSha}`;
+            }
+
+            pdfBody.engineVersion = `${engineVersion}${engineCommitSha && `-${engineCommitSha}`}`;
         }
+
         const config: HttpHeaders = {
             method: 'POST',
             headers: {
                 'Content-Type': 'application/json',
             },
-            body: (documentResponse.data as string) ?? null,
+            body: format !== DownloadFormats.PDF ? (documentResponse.data as string) ?? null : JSON.stringify(pdfBody),

```

Comment:
for this one I'm not sure what it actually does

Reply:
Seems like with `PDF` you can export multiple layouts as you can see `layoutsToExport` so the body when exporting `PDF` is slightly different we add the `layoutsToExport` with the list of layouts ids

Reply:
no I'm specifically talking about the body in config, never saw it before